src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs

```diff
@@ -676,6 +676,66 @@ public interface I
             edits.VerifyRudeDiagnostics();
         }
 
+        [Fact]
+        public void RefStructInsert()
+        {
+            string src1 = "";
+            string src2 = "ref struct X { }";
+
+            var edits = GetTopEdits(src1, src2, CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.Latest));
+
+            edits.VerifyEdits(
+                "Insert [ref struct X { }]@0");
+
+            edits.VerifyRudeDiagnostics(
+                Diagnostic(RudeEditKind.RefStruct, "ref struct X", "struct"));
```

Comment:
>struct [](start = 68, length = 6)

reference to feature resources? here and below

Reply:
Replacing with `SyntaxFacts.GetText(SyntaxKind.StructKeyword)`

---
In reply to: [139556357](https://github.com/dotnet/roslyn/pull/22177#discussion_r139556357) [](ancestors = 139556357)